goLF/new.go

```diff
@@ -2,30 +2,58 @@ package goLF
 
 import (
 	"database/sql"
+	"strconv"
+	"time"
+
+	"github.com/elastic/go-elasticsearch/v8"
+
 	"github.com/LetsFocus/goLF/configs"
 	"github.com/LetsFocus/goLF/database"
+	"github.com/LetsFocus/goLF/elasticstack"
 	"github.com/LetsFocus/goLF/logger"
 )
 
+type Databases struct {
```

Comment:
lets move this to database pkg, create a file named structs.go and keep all the structs of postgres and es inside it